projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -0,0 +1,37 @@
+import { requestAPI } from "./handler";
+
+
+export function getCurrentPathRequest(){
+    requestAPI<any>('run', {
+        method: 'GET',
+    })
+        .then(data => {
+            sessionStorage.setItem("current-path", data["path"]);
+        })
+        .catch(reason => {
+            console.error(
+                `The jlab_vdk server extension appears to be missing.\n${reason}`
+            );
+        });
+}
+
+export function jobRunRequest() {
```

Comment:
do you know about asynccfunctions?
I think they make stuff like this way cleaner